src/plugins/vis_type_drilldown/server/routes/index.ts

```diff
@@ -0,0 +1,17 @@
+import { IRouter } from '../../../../core/server';
+
+export function defineRoutes(router: IRouter) {
```

Comment:
@Willie-The-Lord Do you actually need/want this route, or was this just boilerplate from the template script? I imagine you may be able to get rid of all the `server` aspects of this plugin.

Reply:
It's a boilerplate for the template script.